src/integrationTest/resources/data/non-strategic/birmingham-administrative-court-daily-cause-list/birminghamAdministrativeCourtDailyCauseList.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Can we not use the same test files for all administrative court lists. There are the same list just with different messages in style guide

Reply:
fixed.